docs/aqua-book/language/crdt-maps.md

```diff
@@ -0,0 +1,57 @@
+# CRDT Maps
+
+A map is a kind of [collection](types.md#collection-types) that store arrays of elements that can be accessible by `string` keys. 
```

Comment:
Can we talk about key-value pairs here too?
I think it is also important to say that maps are an appendable collection.
And the link could be given directly to maps section in `types.md`.